ui/nuxeo-document-layout.js

```diff
@@ -30,10 +30,11 @@ import { I18nBehavior } from './nuxeo-i18n-behavior.js';
  *     <nuxeo-document-layout document='{...}' layout="edit"></nuxeo-document-layout>
  *
  * Layouts are stamped by convetion. The href is built based on the document's type (`document.type`)
- * and the layout's name (`layout`). The layout's href is resolved in relation to this elements `importPath`
- * and it follows the following pattern:
+ * and the layout's name (`layout`). The layout's href is resolved in relation to either the parent or
+ * this elements `importPath`, so long as they have `importMeta` defined. The href patter can be configured
```

Comment:
typo: `The href pattern`

Reply:
Was already fixed on a more recent commit.

Reply:
Was already fixed on a more recent commit.